src/common/test-utils/browser.js

```diff
@@ -21,7 +21,35 @@ function testOriginalEvent(spy) {
     expect(spy.getCall(0).args[0].originalEvent instanceof Event).to.equal(true);
 }
 
+/**
+ * Simulates a touch based scroll event over 100ms.
+ *
+ * @param {HTMLElement} el The element to scroll.
+ * @param {number} to The new scrollLeft for the element.
+ */
+function simulateScroll(el, to) {
+    const { scrollLeft } = el;
+    const distance = to - scrollLeft;
+    const duration = 100;
```

Comment:
Was there a specific reason you changed this to 100 from the 10 it was previously? It looks like the failing tests might need higher timeouts to compensate.

Reply:
Yeah I needed a higher duration to be able to cancel the scroll effectively. I'll mess around with the timings.